TuneUp/TuneUpWindowViewModel.cs

```diff
@@ -1187,54 +1187,95 @@ public void ExportToCsv()
 
             if (saveFileDialog.ShowDialog() == true)
             {
-                using (var writer = new StreamWriter(saveFileDialog.FileName))
+                // Check if the .csv file locked or in use
+                if (IsFileLocked(new FileInfo(saveFileDialog.FileName)))
                 {
-                    writer.WriteLine("Execution Order,Name,Execution Time (ms)");
+                    MessageBox.Show("The file is currently in use by another application. Please close the file before trying to overwrite it.",
```

Comment:
@ivaylo-matov can you move these messages to resources so that they will be localized. You can do it in a different PR.

Reply:
Now addressed in this PR : https://github.com/DynamoDS/TuneUp/pull/70